db/migrate/20200625162742_create_comments.rb

```diff
@@ -0,0 +1,12 @@
+class CreateComments < ActiveRecord::Migration[6.0]
+  def change
+    create_table :comments do |t|
+      t.string :author
```

Comment:
- [ ] Remove line # 4 because is making your database specify an author object and you already reference the author in line # 7
![image](https://user-images.githubusercontent.com/52209067/85818757-b3bb5a00-b72e-11ea-9dab-edd0b4a91e81.png)

Reply:
## TSE recommendations applied
✅ Updated readme file
✅ Updated create_comments migration file